homeassistant/components/zwave/__init__.py

```diff
@@ -193,19 +193,19 @@ def _node_object_id(node):
     return node_object_id
 
 
-def _object_id(value):
+def object_id(value):
     """Return the object_id of the device value.
 
     The object_id contains node_id and value instance id
     to not collide with other entity_ids.
     """
-    object_id = "{}_{}_{}".format(slugify(_value_name(value)),
-                                  value.node.node_id, value.index)
+    _object_id = "{}_{}_{}".format(slugify(_value_name(value)),
+                                   value.node.node_id, value.index)
 
     # Add the instance id if there is more than one instance for the value
     if value.instance > 1:
         return '{}_{}'.format(object_id, value.instance)
```

Comment:
The issue might be specifically to do with the `AEOTEC_ZW098_LED_BULB`, as this is the device that is failing for me, and I can see that changes have been made here https://github.com/home-assistant/home-assistant/commit/58f0655298abe2ee169d772f3c75eaeb5d1d9f4d#diff-4f43353f8d1cc611b78ddb29fe62162c relating to this.